tokio/src/runtime/task/waker.rs

```diff
@@ -44,12 +44,38 @@ impl<S> ops::Deref for WakerRef<'_, S> {
     }
 }
 
+cfg_trace! {
+    macro_rules! trace {
+        ($harness:expr, $op:expr) => {
+            if let Some(id) = $harness.id() {
+                tracing::trace!(
+                    target: "tokio::task",
```

Comment:
take it or leave it, but what do you think about giving this a separate target from the one used for task "lifecycle" (spawn/enter/exit/drop) events, so that it can be filtered on/off separately? maybe
```suggestion
                    target: "tokio::task::waker",
```

Reply:
I did wonder specifically about adding `::waker` on the end. I wasn't certain if it'd be better to keep a smaller "target" path, or if it would help to be the same target in the console, and we just for the `op` field. (Really, I wanted the `op` to be the `name`, but 🤷 ).

Or perhaps it's helpful for things even besides the console for someone to say "I'm curious about task logs, but not waker stuff.", and then they could just say something like `RUST_LOG=tokio::task=trace,tokio::task::waker=off`.